modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-report-web/src/main/resources/META-INF/resources/js/components/list/List.es.js

```diff
@@ -27,10 +27,12 @@ export default ({data, field, totalEntries}) => {
 					data.map((field, index) => <li key={index}>{field}</li>)}
 
 				{data.length == 5 && totalEntries > 5 ? (
-					<li key={'see-more'}>
+					<li id="see-more" key={'see-more'}>
```

Comment:
@alinedoleron seria bom botar o portlet namespace como prefixo desse id não? poderias pegar do AppContext, se que tem ainda.

Reply:
se for complicar, não precisa do namespace. será um problema apenas se algum dia o portlet de reports for adicionado a uma pagina que contenha outros portlets.